src/pages/FlagCommentPage.js

```diff
@@ -138,10 +148,15 @@ function FlagCommentPage(props) {
         />
     ));
 
+    const shouldShowLoading = props.isLoadingReportData || props.report.isLoadingReportActions;
```

Comment:
I am a bit concerned that we're using `props.report` directly without confirming if it is undefined? I think it makes sense to add. the following:

```
if (isEmpty(props.report)) {
 Navigation.dismissModal();
return;
}


```

Reply:
I was concerned about non-existent report, but I tested it shows up the invalid access page. Thanks for the patience here.